src/utils/db/car_stream.rs

```diff
@@ -329,6 +329,12 @@ async fn read_v1_header<ReaderT: AsyncRead + Unpin>(
     Some(header)
 }
 
+pub fn uvi_bytes() -> UviBytes {
+    let mut decoder = UviBytes::default();
```

Comment:
`UviBytes::default` cannot be banned in `.clippy.toml` because it does not belong to `UviBytes`